packages/activities/package.js

```diff
@@ -0,0 +1,28 @@
+Package.describe({
+  name: 'wekan:activities',
+  version: '0.0.1',
```

Comment:
We should keep all the package version number in sync with the main app, ie `0.9.0`.